day-zero/catalog-properties.mdx

```diff
@@ -1,5 +1,154 @@
+- **Limited Ability to Enforce Standards:** Without a structured way to categorize resources, enforcing architectural and security standards across the organization becomes difficult.
+
+## How Catalog Properties Help
+
+**Catalog Properties** directly address these challenges by letting you establish a **common language** for describing and managing all resources in your Amplication workspace.
+They let you:
+
+<CardGroup cols={2}>
+
+  <Card title="Establish Strategic Control" icon="gear">
+    Define & enforce organizational standards at scale. Tag each resource with properties that reflect your unique requirements, like domain, environment, or regulatory compliance tier.
+  </Card>
+
+  <Card title="Simplify Maintenance & Reduce Drift" icon="arrows-repeat">
+    Keep resources aligned by leveraging Catalog Properties in automated updates, so you can mitigate technical debt and ensure ongoing compliance.
+  </Card>
+
+  <Card title="Accelerate Developer Velocity" icon="rocket">
+    Free your dev teams from boilerplate and guesswork. Standardize logic and configurations in code generation, allowing developers to focus on critical business logic.
+  </Card>
+
+  <Card title="Enhance Visibility & Governance" icon="eye">
+    Quickly filter, group, and visualize resources in the Service Catalog or the Architecture Graph. Gain insights into how resources are deployed, owned, or integrated.
+  </Card>
+
+</CardGroup>
+
+<Note>
+**Catalog vs. [Blueprint Properties](/day-zero/blueprint-properties-relations)**
+
+Blueprint Properties define the configuration of a specific resource.
+Catalog properties, on the other hand, are **organizational metadata** that apply broadly across all resources for filtering, grouping, governance, and more.
+</Note>
+
+## Smart Code Generation with Catalog Properties
+
+Catalog Properties aren't just for organizing your resources. They also **connect into code generation via plugins**.
+When you generate code using Private Plugins, you have full access to:
+
+- **Catalog Properties** of the specific resource.  
+- **Blueprint Properties** of the resource.  
+- **Catalog & Blueprint Properties** of all related resources set through Blueprint relations.
+
+This means you can **adapt generation logic** based on any property.
+
+- **Compliance Level** triggers the inclusion of stricter security modules.  
+- **Deployment Environment** adjusts Dockerfiles, Helm charts, or serverless config.  
+- **Team Ownership** adds certain logging or monitoring integrations relevant for that team.
+
+<Tip>
+**Private Plugin Property Access**: Catalog Properties are always available in the plugin context for code generation so you can easily maintain enterprise-wide consistency, whether you're generating microservices, infrastructure as code, or messaging configurations.
+</Tip>
+
+## Example Use Cases
```

Comment:
all examples use "select" 
use some other exmaples for text, links, multi-select